scripts/qa_crowbarsetup.sh

```diff
@@ -2410,6 +2410,13 @@ function onadmin_teardown()
 ruby=/usr/bin/ruby
 iscloudver 5plus && ruby=/usr/bin/ruby.ruby2.1
 
+tftpboot_repos_dir=/srv/tftpboot/repos
+tftpboot_suse_dir=/srv/tftpboot/suse-$suseversion
```

Comment:
testing showed that $suseversion is unset here. either we hardcode this here as 11.3 (only Cloud 1.0 had 11.2 and we probably will not be on SP4 ever because of dropped python-2.6 support) or we call onadmin_set_source_variables before.